docs/_posts/2018-07-18-523-release.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: blogpost
+title:  "5.2.3 Release Announcement"
+section_id: blog
+date:   2018-07-18 12:00:00
+author: Open GEE Development Team
+---
+
+Hello Open GEE Community!
+ 
+We are excited to announce the official release of Open GEE 5.2.3!  This release updates Open GEE with minor enhancements and bug fixes for GEE Fusion and Server.
+ 
+Enhancements include:
+
+**Open GEE C++ code now supports and defaults to compiling using the <code>-std=gnu++11</code> compile flag**. This default can be overridden and the C++ code can be compiled using the <code>-std=gnu++98</code> flag instead.
+
+**Remove TCP/IP connections to PostgreSQL.** Open GEE code does, by default, connect to PostgreSQL exclusively using Unix domain sockets. This make it easier to secure PostgreSQL for security sensitive installations.
```

Comment:
Smooth over wording: "Open GEE, by default, connects to PostgreSQL ..."

Should we be more specific about which PostgreSQL instance Open GEE connects to, saying that this is the fusion backend database (or something similar)?

Reply:
fixed